src/components/crossDevice/CrossDeviceSubmit/index.js

```diff
@@ -0,0 +1,51 @@
+import { h, Component } from 'preact'
+
+import { trackComponent } from '../../../Tracker'
+import {preventDefaultOnClick} from '../../utils'
+import theme from '../../Theme/style.css'
+import style from './style.css'
+
+class CrossDeviceSubmit extends Component {
+  constructor (props) {
+    super(props)
+  }
+
+  handleSubmit = () => {
```

Comment:
Minor: I wouldn't bother with defining this function, just call nextStep() directly in render